aw/Extensions/UILabel.swift

```diff
@@ -5,4 +5,19 @@ extension UILabel {
         font = style.font()
         textColor = style.color()
     }
+
+    var isTruncated: Bool {
+
+        guard let labelText = text else {
+            return false
+        }
+
+        let labelTextSize = (labelText as NSString).boundingRect(
```

Comment:
Do you know if doing this actually renders the truncated part of the text? Might be easier to check the number of lines, or some other parameter.

Reply:
It's so that I can check if a label is currently truncated. That way if there is no description, or a short one we can hide the 'Read More' button.

Reply:
Oh, I see. Is there really not a better way to do this, like some way to tell if the system has applied an ellipsis?

Reply:
Not that I found. UILabel.text isn't modified when it's truncated, any changes happen behind the scenes.